tests/umock_c_negt_int/umock_c_negt_int_tests.c

```diff
@@ -329,7 +329,7 @@ TEST_FUNCTION(negative_tests_with_1_call_with_void_ptr_return_dependency)
 TEST_FUNCTION(negative_tests_with_1_call_with_ignored_arguments_takes_the_ignoreargument_flags_into_account)
 {
     size_t i;
-    STRICT_EXPECTED_CALL(function_4_void_ptr_return_non_NULL(IGNORED_PTR_ARG))
+    STRICT_EXPECTED_CALL(function_4_void_ptr_return_non_NULL(IGNORED_ARG))
         .SetFailReturn(NULL).IgnoreArgument(1);
```

Comment:
>IgnoreArgument [](start = 29, length = 14)

Isn't it already ignored at this moment because of IGNORED_ARG?

Oooooh.... how I love typing IGNORED_ARG - I mostly only use my left hand for it :)

Reply:
It's doubly ignored yes, removing one of them.

---
In reply to: [395827151](https://github.com/Azure/umock-c/pull/152#discussion_r395827151) [](ancestors = 395827151)